content/docs/contributing/release-process.md

```diff
@@ -149,7 +149,88 @@ page if a step is missing or if it is outdated.
    example, see
    [upgrading-1.0-1.1](https://cert-manager.io/docs/installation/upgrading/upgrading-1.0-1.1/).
 
-3. Check that the `origin` remote is correct. To do that, run the following
+3. **(final + patch releases)** **Website Updates, part 1** (creating the "release
+   notes" PR).
+
+     **⚠️ This step takes a lot of time, you can do it ahead of time.**
+
+     The steps below need to happen using `master` (**final release**) or
+     `release-1.x` (**patch release**). The PR will be merged after the release.
+
+   1. Generate `release-notes.md` using the instructions further below
+     (<kbd>Ctrl+F</kbd> and look for `github-release-description.md`).
+   2. For each release note entry, open the PR associated and edit the
+      `release-note` block. The editing is about re-writing each entry into 1 to
+      3 sentences written in proper English (subject-verb-complement, ends with
+      a dot). The release note must explain to the end-user whether they should
+      care and how they will be affected. Here is an example of such edit:
+
+      Before:
+      ~~~markdown
+      ```release-note
+      Adds missing comparisons for certain fields which were incorrectly skipped if a LiteralSubject was set
+      ```
+      ~~~
+
+      After:
+      ~~~markdown
+      ```release-note
+      When using the `literalSubject` on a Certificate, the IPs, URIs, DNSNames, and EmailAddresses segments are now properly compared.
```

Comment:
I think this is a generally good thing to do, but probably should be part of a section around PR review process, I don't think we should be modifying release notes for PRs as part of release process.
There is a value in this page contents being concise imo as else there is a risk of skipping parts since it's quite manual copy-paste job rigt now.

Reply:
I think it could go somewhere around here https://cert-manager.io/docs/contributing/contributing-flow/#creating-pull-requests

Reply:
Thanks, I'll move that paragraph.